community/sinit/build

```diff
@@ -1,10 +1,22 @@
 #!/bin/sh -e
 
-make CFLAGS="$CFLAGS -static"
+mv config.h sinit/config.h
 
-install -Dm 755 sinit "$1/usr/bin/sinit"
+cd sinit
+make CFLAGS="$CFLAGS -static" PREFIX=/usr DESTDIR="$1" install
+ln -s /usr/bin/sinit "$1/usr/bin/init"
```

Comment:
Can wrap this in `()` to avoid `cd ..`.

Reply:
In a function like below?

```sh
buildsinit() {
    cd sinit
    make (...) install
    ln -s (...)
}

Reply:
No no,

```sh
(
    cd sinit
    make bla bla
    ln -s bla la
)
```

Then you can avoid `cd ..` as the `cd` is kept local to the braces.

Reply:
Please indent the code like my example above.